src/hipscat/pixel_tree/pixel_tree.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from hipscat.pixel_tree.pixel_node import PixelNode
+
+
+class PixelTree:
+    """Sparse Quadtree of HEALPix pixels that make up the HiPSCat catalog
+
+    This class stores each node in the tree, with leaf nodes corresponding to pixels with data
+    files.
+
+    There are a number of methods in this class which allow for quickly navigating through the
+    tree and performing operations to filter the pixels in the catalog.
+
+    Attributes:
+        pixels: Nested dictionary of pixel nodes stored in the tree. Indexed by HEALPix
+        order then pixel number root_pixel: Root node of the tree. Its children are a subset of the
```

Comment:
```suggestion
             order then pixel number 
        root_pixel: Root node of the tree. Its children are a subset of the
```

Reply:
Indent multi-line descriptions, and line break for second attribute.

Reply:
Thanks! Yeah I need to change my IDE settings to stop auto formatting like that